deployment/postgres-init-db/sql/sched/tables/scheduling_rule.sql

```diff
@@ -0,0 +1,57 @@
+create table scheduling_rule (
+  id integer generated always as identity,
+  revision integer not null default 0,
+  definition jsonb not null,
+
+  model_id integer not null,
+  description text null,
+  author text null,
+  last_modified_by text null,
+  created_date timestamptz not null default now(),
+  modified_date timestamptz not null default now(),
+
+  constraint scheduling_rule_synthetic_key
+    primary key (id)
+-- TODO `references mission_model` disabled for now, must refer to `aerie_merlin` DB
+--  constraint scheduling_rule_references_mission_model
+--    foreign key (model_id)
+--      references mission_model
+--      on update cascade
+--      on delete cascade
+);
+
+comment on table scheduling_rule is e''
+  'A goal or rule for scheduling of a plan.';
```

Comment:
We can just have "A rule for ..." you seem to use "rule" everywhere else.

Reply:
Good catch! Will do